### ISSUE TITLE
simplify access to diagrams (and other items in drop down menu)

### ISSUE THREAD

**Issue by TheOtherDoctor** [martykan/forecastie#468]
I often want to look into the diagrams for the next days, which I find a very useful feature. 
On my device, the top right icons are refresh, search, map and the three-dots-more menu. 
Although there is a lot of free space in the top header line, I have to go via the "..." drop down to get  to diagrams. On the other hand, I use none of the top level icons in the title bar.

So for me, the usability could be easily improved by putting the menu points from the "..." drop down just also in the header bar.

Version is 1.13.5 from f-droid; running on Lineage 14.1; 
Device has 1280x800, even in portrait with only 800pix width there is lots of unused space. 

Side note: In the bottom ("today", "tomorrow", ...) region, in landscape mode there is so much horizontal space (>50% of screen width) that I just wonder why not layouting all data info simply into one table, removing any need for scrolling entirely...

**Comment by robinpaulson** [comment]
> I often want to look into the diagrams for the next days, which I find a very useful feature.
> On my device, the top right icons are refresh, search, map and the three-dots-more menu.
> Although there is a lot of free space in the top header line, I have to go via the "..." drop down to get to diagrams. On the other hand, I use none of the top level icons in the title bar.
> 
> So for me, the usability could be easily improved by putting the menu points from the "..." drop down just also in the header bar.
> 

I presume by "Diagrams" you mean "Graphs"?

Check the next release, the issue we've been working on for the last week or so will cover this in a slightly different way: #456 . It'll be out on F-Droid today or tomorrow

> Side note: In the bottom ("today", "tomorrow", ...) region, in landscape mode there is so much horizontal space (>50% of screen width) that I just wonder why not layouting all data info simply into one table, removing any need for scrolling entirely...

Yes, true. It would probably require a lot of fiddling around with layouts, so not trivial. It's an interesting suggestion though. Are you capable of contributing code for part or all of it? If so, go for it.

**Comment by robinpaulson** [comment]
We could also test for screen size, then show everything in one go if it's bigger than a certain size, no swiping required. Useful and worthwhile, but low priority I'm afraid.

**Comment by TheOtherDoctor** [comment]
> I presume by "Diagrams" you mean "Graphs"?

Yes, I had to guess from my language locale what the english entry might be. ;-)

> Check the next release [...] #456

Meanwhile I had run across the graphs button "quick access" functionality, but actually only by reading this in the closed issue here (#300) - thus I'd totally confirm that putting a (splash-like one time) hint on this is a very good idea; obviously I just gave another proof that there are users who, although searching for this, do not consider this icon might be a button. ;-) 
Once known, the solution with clicking the icon is perfect, way easier to hit this large icon than a tiny menu item. 

Thanks for changing the issue title to the remaining open question;
> why not layouting all data info simply into one table [...] Are you capable of contributing code for part or all of it? If so, go for it.

Unfortunately my coding skills clearly do not yet cover this yet. :-(
